ui/perfherder/helpers.js

```diff
@@ -186,8 +186,11 @@ export const getCounterMap = function getCounterMap(
     cmap.originalRuns = orig.runs;
     cmap.originalStddev = orig.stddev;
     cmap.originalStddevPct = orig.stddevPct;
+    cmap.originalJobIds = originalData.job_ids;
```

Comment:
Let's store just a single job id here. Then update CompareTable.retrigger to use a more precise first parameter.